Makefile

```
@@ -18,14 +18,20 @@ makefiles = \
   src/libexpr/local.mk \
   src/libcmd/local.mk \
   src/nix/local.mk \
+  src/libutil/c/local.mk \
+  src/libstore/c/local.mk \
+  src/libexpr/c/local.mk \
```

```suggestion
  src/libutil-c/local.mk \
  src/libstore-c/local.mk \
  src/libexpr-c/local.mk \
```
I have been trying to keep the structure flat here (for source filtering reasons, and preparation for Meson)

---

Done in https://github.com/NixOS/nix/pull/8699/commits/c57de60522c3f2d493b6c013072b5c2b0dea3f0d